src/vcpkg.cpp

```diff
@@ -41,6 +41,53 @@ static void invalid_command(const std::string& cmd)
     Checks::exit_fail(VCPKG_LINE_INFO);
 }
 
+static bool detect_container(vcpkg::Filesystem& fs)
+{
+    (void)fs;
+#if defined(_WIN32)
```

Comment:
```suggestion
#if defined(_WIN32)
    (void)fs;
```

Reply:
OSX CI doesn't like unused parameters, so I left it out of the `#if` block

Reply:
Ah, that's not #else, OK!